src/regression_tests/regression_test.test.ts

```diff
@@ -0,0 +1,137 @@
+const ADMIN_XPUB =
+  'xpub661MyMwAqRbcFgfmdkPgE2m5UjHXu9dj124DbaGLSjaqVESTWfCD4VuNmEbVPkbYLCkykwVZvmA8Pbf8884TQr1FgdG2nPoHR8aB36YdDQh';
+
+let paymailDomainInstanceOne: string = '';
+let paymailDomainInstanceTwo: string = '';
+let userOne: RegressionTestUser;
+let userTwo: RegressionTestUser;
+let rtConfig: RegressionTestConfig;
+
+beforeAll(() => {
+  fetchMock.disableMocks();
+  rtConfig = getEnvVariables();
+  expect(rtConfig).toBeDefined();
+});
+
+afterAll(async () => {
+  if (userOne) {
+    await expect(removeRegisteredPaymail(userOne.paymail, rtConfig.clientOneURL, ADMIN_XPRIV)).resolves.not.toThrow();
+  }
+
+  if (userTwo) {
+    await expect(removeRegisteredPaymail(userTwo.paymail, rtConfig.clientTwoURL, ADMIN_XPRIV)).resolves.not.toThrow();
+  }
+});
+
+describe('TestRegression', () => {
+  describe('Initialize Shared Configurations', () => {
+    test('Should get sharedConfig for instance one', async () => {
+      paymailDomainInstanceOne = await getPaymailDomain(ADMIN_XPRIV, rtConfig.clientOneURL);
+      expect(paymailDomainInstanceOne).not.toBe('');
+    });
+
+    test('Should get sharedConfig for instance two', async () => {
+      paymailDomainInstanceTwo = await getPaymailDomain(ADMIN_XPRIV, rtConfig.clientOneURL);
+      expect(paymailDomainInstanceTwo).not.toBe('');
+    });
+  });
+
+  describe('Create Users', () => {
+    test('Should create user for instance one', async () => {
+      const userName = 'instanceOneUser1';
+      userOne = await createUser(userName, paymailDomainInstanceOne, rtConfig.clientOneURL, ADMIN_XPRIV);
+    });
+
+    test('Should create user for instance two', async () => {
+      const userName = 'instanceTwoUser1';
+      userTwo = await createUser(userName, paymailDomainInstanceTwo, rtConfig.clientTwoURL, ADMIN_XPRIV);
+    });
+  });
+
+  describe('Perform Transactions', () => {
+    test(
+      'Send money to instance 1',
+      async () => {
+        const amountToSend = 3;
+
+        const transaction = await sendFunds(
+          rtConfig.clientTwoURL,
+          rtConfig.clientTwoLeaderXPriv,
+          userOne.paymail,
+          amountToSend,
+        );
+        expect(-1).toBeGreaterThanOrEqual(transaction.output_value);
+
+        const balance = await getBalance(rtConfig.clientOneURL, userOne.xpriv);
+        expect(balance).toBeGreaterThanOrEqual(1);
+
+        const transactions = await getTransactions(rtConfig.clientOneURL, userOne.xpriv);
+        expect(transactions.length).toBeGreaterThanOrEqual(1);
+      },
+      TEST_TIMEOUT_MS,
+    );
+
+    test(
+      'Send money to instance 2',
+      async () => {
+        const transaction = await sendFunds(
+          rtConfig.clientOneURL,
+          rtConfig.clientOneLeaderXPriv,
+          userTwo.paymail,
+          MINIMAL_FUNDS_PER_TRANSACTION,
+        );
+        expect(-1).toBeGreaterThanOrEqual(transaction.output_value);
```

Comment:
Maybe let's reverse this condition, to be more "self-descriptive":
`expect(transaction.output_value).toBeLessThanOrEqual(-1);`

Reply:
Yeah I had that in the first draft but decided to change it to match the go code. Sure thing, I will reverse it